app/controllers/factories_controller.rb

```diff
@@ -18,9 +18,9 @@ def craft
     if params[:loader] && params[:type] && params[:amount] && current_user.location.industrial_station?
 
       if params[:type] == 'ship'
-        ressources = Spaceship.ship_variables[params[:loader]]['crafting'] rescue nil
+        ressources = Spaceship.get_attribute(params[:loader], :crafting) rescue nil
       elsif params[:loader].include?('equipment.')
-        ressources = get_item_attribute(params[:loader], 'crafting')
+        ressources = Item.get_attribute(params[:loader], 'crafting')
```

Comment:
Could that also be changed to `:crafting` ?

Reply:
same as above :)